spec/requests/acuant_sdk_spec.rb

```diff
@@ -2,47 +2,41 @@
 
 describe 'requesting acuant SDK assets' do
   context 'with a valid Acuant SDK asset' do
-    it 'renders a JS asset' do
-      get '/verify/doc_auth/AcuantImageProcessingWorker.min.js'
+    it 'renders a JS asset, without CSP headers or session cookie' do
+      get '/acuant/11.4.1/AcuantImageProcessingWorker.min.js'
 
       expect(response.status).to eq(200)
       expect(response.headers['Content-Type']).to eq('application/javascript')
+      expect(response.headers['Content-Security-Policy']).to be_nil
```

Comment:
I was thinking in a different way to expect `Content-Security-Policy` is not part of the headers without using the fact it will be nil if it doesn't (a ruby knowledge). On maintainers perspective it _reads_[1] more natural

WDYT about this?

```suggestion
      expect(response.headers).to_not have_key('Content-Security-Policy')
```

[1] completely subjective.

Reply:
I like this, thanks

Reply:
Added in 6ab36e342138af3f4eba7765ecbcf5f81f6568a3